### PR TITLE
Items in Search Now Have Dates

### LINKED ISSUE #3615: in search results, registrations of the same project cannot be distinguished from one another
from the community repo: https://github.com/CenterForOpenScience/community/issues/276

## Steps
1. Search the OSF for a project that has been registered multiple times (try searching "local versus global")

## Outcome
1. Registrations of the same project are listed as separate results
2. The registrations are indistinguishable from one another
3. They do not necessarily appear next to each other or in recent to oldest order.
screen shot 2015-07-15 at 10 54 27 am

## Suggested Behavior
1. Registrations should be distinguishable from one another.
2. They do not have to be listed as separate results. One suggestion is to collapse registrations of the same project that are returned in search results into one result, with the option to expand that result. On expansion, the different registrations and their dates could be shown and linked to. If this model is followed, I would suggest making the link visible when unexpanded (the title of the registration int he image below) link to the project's registrations page, so that the user is brought to a place where they can see each registration listed.

screen shot 2015-07-15 at 11 06 59 am

Of course, this could almost certainly be improved visually, but the basic idea is there.

### PR DESCRIPTION
**Pull request by Johnetordoff**
<h1> Purpose </h1>
Allows users to distinguish projects and registrations with the same name. Close #3615.
<h1> Changes </h1>
small js changes to format dates and html to display them.
<h1> Side Effects </h1>
None.

### CODE REVIEW

File: website/templates/search.mako [hunk 1]
```diff
@@ -203,6 +203,7 @@
         <h4><span data-bind="if: parent_title">{{ parent_title }} /</span> <a data-bind="attr.href: url">{{title }}</a></h4>
         <!-- /ko -->
 
+        <strong><span data-bind="text: 'Date Created: ' + date_created['local'], tooltip: {title: date_created['utc']}" {{ date_created['local'] | fit:500 }}></span></strong>
```
Comment (sloria) [inline website/templates/search.mako]: Why is `{{ date_created['local'] | fit:500 }}` within the span tag?

Comment (Johnetordoff) [inline website/templates/search.mako]: That's just a mistake, sorry!

Comment (Johnetordoff) [inline website/templates/search.mako]: Sorry this whole thing is an old commit I need to push again. Hold off a for a while. Sorry again!

**Comment by lbanner** [comment]
@Johnetordoff ... adding dates to 'Project names' might be out of scope re: #3615. In that case we are only talking naming of Registrations of Projects. Is it possible limit the addition of dates to names to only Registrations?

**Comment by Johnetordoff** [comment]
Yes, but I don't see the harm in including dates in both, theoretically if you named two project the same thing the date would be the best identifier. I figured people are likely to accidentally make a bunch of duplicate projects and then other people will need to pick out the duplicates from the original.

**Comment by lbanner** [comment]
Might be true -- but that's an additional, or larger change in practice/OSF functionality needing Community discussion and OK. Will ping @jlcohoon here on this ... I will try to get us clear direction and opinion.

**Comment by jlcohoon** [comment]
I think that dates are only needed for registrations. 
My reason being that the key distinguishing feature of a registration is its date, so it makes sense to identify the date in search results. A project should be distinguished by its content and we already use breadcrumbs and descriptions to give that kind of context in search results.

**Comment by lbanner** [comment]
@Johnetordoff -- with @jlcohoon's note above, can you re-narrow this to dates in names of Registrations only? Thanks!

**Comment by Johnetordoff** [comment]
@lbanner no problem

**Comment by Johnetordoff** [comment]
I improperly merged this branch improperly and decided to just create a new one it is here: #3911

**Pull request closed without merging**